terraform/modules/scheduled_pipelines/scheduled_jobs.auto.tfvars.example

```diff
@@ -17,7 +17,6 @@
 cloud_schedulers_config = {
```

Comment:
why are names removed?

Reply:
using the key instead - also using this for `display_name` of the PipelineJob